docs/development/mdx-reference.md

```diff
@@ -181,21 +181,21 @@ graph LR;
 	Quad -- Smart Audio --> VTX;
 ```
 
-## Latex Equations
+## $\LaTeX$ Equations
```

Comment:
This does not render right

![image](https://github.com/betaflight/betaflight.com/assets/8344830/5126288a-67da-4f7b-8455-8ce9f5c03cfe)

Reply:
What is not rendered right? Header 3?

Reply:
??

`aedb7604e` :
![image](https://github.com/betaflight/betaflight.com/assets/56646290/7de80deb-ca99-450e-acf2-fc11bf6407ba)

Reply:
[deleted my comment/screenacap, i see it now.]